mail-filter/mimedefang/mimedefang-2.84.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils user
+
+DESCRIPTION="Antispam, antivirus and other customizable filters for MTAs with Milter support"
+HOMEPAGE="http://www.mimedefang.org/"
+SRC_URI="http://www.mimedefang.org/static/${P}.tar.gz"
+LICENSE="GPL-2+"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="clamav +poll"
+
+DEPEND=">=dev-perl/MIME-tools-5.412
+	dev-perl/IO-stringy
+	virtual/perl-MIME-Base64
+	dev-perl/Digest-SHA1
+	dev-perl/MailTools
+	dev-perl/Unix-Syslog
+	clamav? ( app-antivirus/clamav )
+	|| ( mail-filter/libmilter mail-mta/sendmail )"
+RDEPEND="${DEPEND}"
+RESTRICT="test"
```

Comment:
Restricting tests requires a comment justifying the restriction. And yes, that implies you should at least try to get tests working.